### DISCUSSION TITLE
Change the background color of themes

### DISCUSSION THREAD

**Discussion by Mailiax** [picocss/pico#311] [Help]
Maybe I am just stupid, but I can't find a way to change the background colors of the dark and light themes. 

Shouldn't you just be able to import pico in an scss file, change  `--background-color: ` in the mixin, and then compile it to css and import it into the project? This doesn't work for me. The resulting css file does not reflect the change of `--background-color: ` in the scss file.

**Comment by sgirard84** [comment]
This works in custom css file

```
:root {
  background-color: red !important;
}
```

Obviously you can set it for light and dark themes.
